app/assets/javascripts/modules/moj.sentry.js

```diff
@@ -0,0 +1,18 @@
+(function() {
+  'use strict';
+
+  moj.Modules.Sentry = {
+    el: '.js-Sentry',
+    init: function() {
+      this.raven = Raven;
+      this.sentry_js_dsn = $(this.el).data('sentry-js-dsn');
+      this.raven.config(this.sentry_js_dsn).install();
+      var self = this;
```

Comment:
Because self is already defined on the ```window``` object and could potentially break stuff.
There is a safe way to do this in javascript by using ```.bind()```

https://www.w3schools.com/jsref/prop_win_self.asp